planner/util/path.go

```diff
@@ -50,6 +50,9 @@ type AccessPath struct {
 
 	IsDNFCond bool
 
+	// IsGlobalRead indicates whether this path is a remote read path for tiflash
+	IsGlobalRead bool
```

Comment:
IsGlobalRead is not obvious, it is not related to tiflash.

Reply:
Done, changed to IsTiFlashGlobalRead